wnfs-common/Cargo.toml

```diff
@@ -28,12 +28,14 @@ once_cell = "1.16"
 proptest = { version = "1.1", optional = true }
 rand_core = "0.6"
 serde = { version = "1.0", features = ["rc"] }
+serde_json = "1.0"
 thiserror = "1.0"
 
 [dev-dependencies]
 async-std = { version = "1.11", features = ["attributes"] }
 proptest = "1.1"
 rand = "0.8"
+tempfile = "3.5"
```

Comment:
Where is this used? Can this be removed?

Reply:
removing